docs/changes.rst

```diff
@@ -4,6 +4,27 @@ Change log
 1.6 (upcoming)
 --------------
 
+Removed features
+~~~~~~~~~~~~~~~~
+
+* Support for automatic setup has been removed. The automatic setup was
```

Comment:
I only expected something like "as it was frequently problematic...." without all the detailed bullet points. It might be useful to include these details on the issue in case someone complains and wants to see the detailed justification, but it's not so important to mention in the release notes, I think.

Although, I have a small/medium django-cms project that seems to use the automatic setup (not sure I'd describe as a "trivial project". ;-)